### PR TITLE
chore: Migrate extensions into CCA (singleline-extension)

### PR DESCRIPTION
**Pull request by alexdonets**
#### Part 5/5 - `singleline-extension`

This PR is the last of 5 PRs which implement the migration of the extensions from CCE into CCA.
In this PR we are migrating the `singleline-extension`.

#### Extensions migrated to CCA
- [x] `entry-extension-collapsible`
- [x] `markdown-extension`
- [x] `multiple-references-extension`
- [x] `rich-text-extension`
- [x] `singleline-extension`

#### To Do
 - [x] Update build scripts with `cross-env` in all new apps.

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Example extensions
+# Example apps
 
 This folder contains example extensions that you can easily build using `@contentful/field-editor-*` packages.
 
-All these extensions are build with [Create Contentful Extension CLI][create-contentful-extension]
+All these extensions are build with [Create Contentful App CLI][create-contentful-app]
```
Comment (Jwhiles) [inline extensions/README.md:5]:
```suggestion
All these extensions are built with [Create Contentful App CLI][create-contentful-app]
```

**Review by Jwhiles — APPROVED** [review]
Looks good!

**Review by shikaan — APPROVED** [review]
nit: I would call the folder cointaining all the apps "apps" rather than extensions and generally speaking change all the references from extensions to apps

**Review by shikaan — APPROVED** [review]
🧩

**Pull request merged**